projects/libs/core-library-typescript/package.json

```diff
@@ -72,6 +75,7 @@
     "crc": "^3.5.0",
     "ganache-core": "^2.0.2",
     "mocha": "^5.0.0",
+    "request-promise": "^4.2.2",
```

Comment:
Where do we use it?

Reply:
Removed express,request-promise from devDependencies.
rm -rf dist & rm -rf distTree are useful to remove junk on build and they are very fast.